modules/programs/mcfly.nix

```
@@ -30,8 +30,19 @@ in {
     enableFuzzySearch = mkOption {
       default = false;
       type = types.bool;
+      description = ''
+        Whether to enable fuzzy searching.  This option is deprecated,
+        corresponding to a fuzzySearchFactor of 2 (true) or 0 (false).
+      '';
+    };
+
+    fuzzySearchFactor = mkOption {
+      default = 0;
+      type = types.int.positive;
```

Needs to be
```suggestion
      type = types.int.unsigned;
```
to allow 0.